### PR TITLE
Make invoker goals overridable on the CLI

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,6 +74,12 @@
         <rpkgtests-maven-plugin.version>0.8.0</rpkgtests-maven-plugin.version>
         <groovy-maven-plugin.version>2.1.1</groovy-maven-plugin.version>
         <useReleaseProfile>true</useReleaseProfile>
+
+        <!-- To run native tests, use
+
+             mvn clean verify -Dinvoker.goals="clean verify -e -Dnative"
+        -->
+        <invoker.goals>clean verify -e</invoker.goals>
```
Comment (aloubyansky) [inline pom.xml:82]: I think I can define `native` profile that will override the property. Then simply `-Pnative` will be enough.
Have to you tried cleaning your repo from all the universe artifact including the rpkgtests artifacts and running `clean verify`?

Comment (ppalaga) [reply pom.xml:82]: > I think I can define `native` profile that will override the property.

+1, but please still leave `invoker.goals` overridable via CLI

**Comment by ppalaga** [comment]
Feel free to close this one if you do not plan to merge it, @aloubyansky

File: pom.xml (hunk 1 above)
Comment (aloubyansky) [reply pom.xml:82]: `verify` works. I remember trying it before and got resolution failures. I guess it was something else.

**Comment by aloubyansky** [comment]
Ok, I added the profile and info about running tests to the readme. I'm gonna close this one. Thanks for the ideas though.

**Pull request closed without merging**